package.json

```diff
@@ -26,30 +26,30 @@
   "dependencies": {
     "@discordjs/builders": "^0.8.2",
     "@discordjs/rest": "^0.1.0-canary.0",
-    "@types/node": "^16.9.1",
-    "@types/node-fetch": "2.5.12",
+    "@types/node": "^16.11.6",
+    "@types/node-fetch": "v2",
     "chalk": "^4.1.2",
     "discord-api-types": "^0.24.0",
-    "discord.js": "^13.1.0",
+    "discord.js": "^13.3.1",
     "dotenv": "^10.0.0",
     "moment": "^2.29.1",
-    "node-fetch": "2.6.2",
+    "node-fetch": "v2",
```

Comment:
What's this? Why just `v2` now?

Reply:
`node-fetch` v3 is not compatible with our project as it is ES module only...
It was v2 already, but I just made it v2 in `package.json` to really keep it at v2.

https://github.com/node-fetch/node-fetch/blob/main/docs/CHANGELOG.md#v300

v2 still being update tho :)